challenges-services/src/main/resources/db/changelog/challenge-rdbms.db.changelog-1.0.0.xml

```diff
@@ -114,4 +114,16 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.-->
         <addAutoIncrement columnDataType="BIGINT"  columnName="ANNOUNCEMENT_ASSIGNEE_ID"  incrementBy="1" startWith="1" tableName="ANNOUNCEMENT_ASSIGNEE"/>
     </changeSet>
 
+    <changeSet author="challenge" id="1.0.0-6">
+        <addColumn tableName="CHALLENGE">
+            <column name="POINTS" type="BIGINT" defaultValueNumeric="20"/>
```

Comment:
Let's put the defazult value in the service layer to be able to change the default value there

Reply:
I added this here to initialize the old challenges with the value 20